src/librustc_back/target/apple_base.rs

```diff
@@ -37,6 +37,7 @@ pub fn opts() -> TargetOptions {
         function_sections: false,
         dynamic_linking: true,
         executables: true,
+        target_family: Some("unix".to_string()),
```

Comment:
Everything that isn't windows, is set to Unix. That includes Linux, OS X, iOS, NetBSD, OpenBSD, FreeBSD, Bitrig, Solaris, and Haiku